src/Components/__tests__/Accordion.spec.js

```diff
@@ -0,0 +1,34 @@
+import react from 'react';
+import {shallow} from 'enzyme';
+import { Accordion, AccordionSummary, AccordionDetails } from '@mui/material';
+
+import Text from '../../ui-kit/Text';
+import FAQAccordian from '../Accordion';
+
+describe("<FAQAccordian />", () => {
+    let wrapper;
+    beforeEach( () => {
+        wrapper = shallow(<FAQAccordian />)
+    });
+
+    describe("ui", () => {
+
+        it("renders accordian elements", () => {
+            expect(wrapper.find(Accordion)).not.toHaveLength(0);
```

Comment:
This is totally a personal preference of mine, but I like to write "positive" tests instead of "negative" tests. For example, if we're testing that something does `not have length 0` that basically means we're testing if something exists, right? To my mind, I would probably test that 
`expect(wrapper.exists(Accordion)).toEqual(true);` (though this may not be valid enzyme syntax, I'm not really sure). Again, this is just a personal preference, so go with what makes the most sense to you!

Reply:
I do like to do positive as well. I was debating between that and gauging a changing number of items depending on how many are used in the FAQ. But they test practically the same thing so the number won't matter. I will update it.